docs/devguide/create-metricset.asciidoc

```diff
@@ -17,7 +17,7 @@ To create a new metricset:
 make create-metricset
 ----
 +
-You need to use Python 2.7 to run this command. If you also have Python 3 in your system, you can run the command with `VIRTUALENV_PARAMS="-p python2.7" make create-metricset` to force it to run with Python 2.7.
+You need to use Python 2.7 to run this command. If you also have Python 3 in your system, you can run the command with +VIRTUALENV_PARAMS="-p python{python}" make create-metricset+ to force it to run with Python 2.7.
```

Comment:
There are still a couple of places where you say "Python 2.7"

Reply:
Note that `{python}` is defined as:

:python: 2.7.9

So `python{python}` resolves as `python2.7.9`. If you need this to resolve to a shorter version (2.7), you''ll need to add another attribute to the versions.asciidoc file and use that attribute here.

Reply:
I messed up here the python titles with commands. I think now with a new parameter in the versions file it should be more clear